cmd/build.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"context"
+	"log"
+	"os"
+	"path"
+	"time"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/screwdriver-cd/sd-local/buildlog"
+	"github.com/screwdriver-cd/sd-local/config"
+	"github.com/screwdriver-cd/sd-local/launch"
+	"github.com/screwdriver-cd/sd-local/screwdriver"
+	"github.com/spf13/cobra"
+)
+
+const (
+	waitIO = 1
+)
+
+func newBuildCmd() *cobra.Command {
+	buildCmd := &cobra.Command{
+		Use:   "build [job name]",
+		Short: "Run screwdriver build of the specified job name.",
+		Long:  `Run screwdriver build of the specified job name.`,
+		Args:  cobra.MinimumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			homedir, err := homedir.Dir()
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			config, err := config.ReadConfig(path.Join(homedir, ".sdlocal", "config"))
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			api, err := screwdriver.New(config.APIURL, config.Token)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			job, err := api.Job(args[0], "./screwdriver.yaml")
```

Comment:
I think it is better to use `os.Getwd()` for using current dir same as [L53](https://github.com/screwdriver-cd/sd-local/pull/8/files#diff-ce0151979a0da3cf1233dd0fd63a5aa8R53).

Reply:
thx
I fixed it